WooCommerce/Classes/POS/Presentation/ProductGridView.swift

```diff
@@ -9,7 +9,7 @@ struct ProductGridView: View {
 
     var body: some View {
         let columns: [GridItem] = Array(repeating: .init(.fixed(120)),
-                                        count: viewModel.products.count)
+                                        count: Constants.maxItemsPerRow)
```

Comment:
Quick update to limit the products per row, otherwise we would show all of them in the same row, pushing the Cart view out of visible space.